public/app/features/canvas/elements/icon.tsx

```diff
@@ -115,6 +115,7 @@ export const iconItem: CanvasElementItem<IconConfig, IconData> = {
         editor: ResourceDimensionEditor,
         settings: {
           resourceType: 'icon',
+          maxFiles: 2000,
```

Comment:
can we maybe export this somewhere as we are using two places?

Reply:
Good idea, was thinking the same! Do you have a suggestion on where this could live?

Reply:
hm maybe as a const `FolderPickerTab` (the furthest down level where we are actually using this value for the time being?)